tests/guarded_transmute_pod_pedantic/mod.rs

```diff
@@ -1,15 +1,15 @@
-use safe_transmute::{Error, ErrorReason, GuardError, guarded_transmute_to_bytes_pod_many, guarded_transmute_pod_pedantic};
+use safe_transmute::{Error, ErrorReason, GuardError, safe_transmute_to_bytes, safe_transmute_one_pedantic};
 
 
 #[test]
 fn too_short() {
-    assert_eq!(guarded_transmute_pod_pedantic::<u32>(guarded_transmute_to_bytes_pod_many::<u32>(&[])),
+    assert_eq!(safe_transmute_one_pedantic::<u32>(safe_transmute_to_bytes::<u32>(&[])),
```

Comment:
Module name doesn't match tested funxion – are we letting this slide 'till the test reorganisation?